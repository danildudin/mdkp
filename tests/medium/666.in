28 9
909 867 901 901 714 927 267 122 695 925 288 620 929 751 658 765 262 101 559 505 567 48 847 984 864 746 321 354 
142 787 582 451 332 424 781 277 589 956 181 98 57 138 755 707 182 216 927 215 789 82 12 16 523 538 747 179 8102
585 796 273 775 214 624 770 998 679 756 66 531 530 170 170 938 213 387 794 904 161 501 887 543 402 314 104 991 7273
501 9 823 41 520 862 805 982 627 486 555 58 356 772 368 184 249 252 583 580 586 308 957 215 58 93 849 835 6883
406 267 994 676 131 15 348 203 209 516 147 859 477 290 30 320 307 947 747 631 990 283 388 919 784 977 881 954 9990
216 213 858 445 944 925 988 880 513 203 724 646 866 436 661 215 196 479 96 576 219 672 405 801 160 200 902 94 9946
200 392 126 171 648 310 788 112 194 814 799 456 903 638 846 130 651 643 2 456 103 642 484 615 597 895 194 13 9375
870 41 454 555 729 40 216 691 782 715 133 267 489 664 115 835 406 790 475 576 455 920 157 407 109 634 581 956 5505
570 676 251 534 885 813 668 118 562 953 182 895 390 926 853 956 224 551 172 726 432 482 370 111 227 69 419 14 5751
667 291 800 856 446 759 230 344 505 968 592 80 180 930 632 472 67 851 628 927 842 221 561 710 900 437 53 927 5503
