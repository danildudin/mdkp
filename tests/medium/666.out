9214
12
0 1 2 3 5 11 12 16 22 23 25 26 
